ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -121,12 +133,14 @@ open class TwoLineTitleView: UIView {
 
     @objc public weak var delegate: TwoLineTitleViewDelegate?
 
+    private var alignment: Alignment = .center
     private var interactivePart: InteractivePart = .none
+    private var animatesTextColor: Bool = true
```

Comment:
its bit unclear to me what animate text color means?

Reply:
If you tap the title view in the "Objective-C Demos" section of the iOS test app, the text "flashes" briefly to indicate a press. Our existing "large title" doesn't flash, so we're adding the option to disable the flashing.

![Simulator Screen Recording - iPhone 14 Pro - 2023-02-01 at 12 28 08](https://user-images.githubusercontent.com/717674/216156483-87f88b6a-fc01-40fc-aafd-a60f3986340a.gif)

Reply:
Perhaps a better name would be something like `textFlashesWhenPressed`.